charts/testkube-operator/templates/tests.testkube.io_testsuites.yaml

```diff
@@ -901,9 +901,6 @@ spec:
                               scraperTemplateReference:
                                 description: scraper template extensions reference
                                 type: string
-                              sync:
-                                description: whether to start execution sync or async
-                                type: boolean
```

Comment:
will previous CRDs work, if they still have it?

Reply:
good point, I have to test this

Reply:
unfortunately, the old CRDs don't work with the new version of the code. thanks for pointing it out, I will revert the CRD changes, and solve this differently